Makefile

```diff
@@ -216,6 +216,7 @@ lint:
 lint-metrics:
 	hack/dockerized "./hack/prom-metric-linter/metrics_collector.sh > metrics.json"
 	./hack/prom-metric-linter/metric_name_linter.sh --operator-name="kubevirt" --sub-operator-name="kubevirt" --metrics-file=metrics.json
```

Comment:
You could create a [trap](https://tldp.org/LDP/Bash-Beginners-Guide/html/sect_12_02.html)  inside `metric_name_linter.sh` so it will clean `metrics.json` file if the file exists. Here's a [reference](https://github.com/kubevirt/kubevirt/blob/ad833ec6935c4c5fd112526c7bae2b8abd1a495f/automation/perfscale-test.sh#L25).

Reply:
It re-creates it if the file exists, still needed?